classes/Context.php

```diff
@@ -514,7 +514,7 @@ public function getComputingPrecision()
     {
         if ($this->priceComputingPrecision === null) {
             $computingPrecision = new ComputingPrecision();
-            $this->priceComputingPrecision = $computingPrecision->getPrecision($this->currency->precision);
+            $this->priceComputingPrecision = $computingPrecision->getPrecision(isset($this->currency->precision)?$this->currency->precision:2);
```

Comment:
```suggestion
            $this->priceComputingPrecision = $computingPrecision->getPrecision($this->currency->precision ?? 2);
```

Reply:
Is the default value `2` chosen arbitrary?

Reply:
Hi,
Sorry for late.
@lartist 
The value 2 is because when you create new currency, by default provide the 2 value into "Decimals" field. 

@kpodemski 
About this solution, is possible that external modules use this function for any reason. 
I was checking an error in a website , and found this error because a module was using it, and i was getting a 500 error when a new customer sign up.


In short , 
This won't fix any error for prestashop core , but will help that if for any reason this field is empy, won't crash.

If you want i can create a ticket more detailed the wednesday. (I can't get all info right now)

Thanks.

Reply:
I think we could introduce a fallback in that method, but the right solution would be for modules to use proper currency object. 🤔